lametro/events.py

```diff
@@ -139,7 +145,7 @@ def _merge_events(self, events):
 
         except AssertionError:
             unpaired_events = '\n'.join(str(web_event) for _, web_event in spanish_events.values())
-            raise AssertionError('Unpaired Spanish events remain:\n{}'.format(unpaired_events))
+            logging.warning('Unpaired Spanish events remain:\n{}'.format(unpaired_events))
```

Comment:
let's do the same message format for each event in `unpaired_events`: `Cannot find English companion for {EventBodyName} at {EventTime} on {EventDate}. {api_url}`.

Reply:
hm, thinking about this, it seems like the scraper should behave consistently when it cannot pair spanish events. that is, it should break. in the past, if something violates our assumptions (usually [duplicate bills in chicago](https://github.com/opencivicdata/scrapers-us-municipal/blob/06b22cbcaae7322b009cd9a87a7c2b12e501348e/chicago/bills.py#L133)), we have skipped over the offending data. what do you think, @jmithani? should we raise an exception here, unless it's the one event we know is unpaired?

Reply:
good thing to flad! bc of the idea of scrapers being "brittle" i think we should continue to have it break. we can create a more useful error message following the format of the other error message you drafted.